recipes/jigsawpy/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "jigsawpy" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/dengwirda/jigsaw-python/archive/v{{ version }}.tar.gz
+  sha256: 36901576cee361d8ee0d2977e9022626b37801f47aebd234f8a008c74557e711
+  patches:
+    - 0001-add-find-library.patch
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools
+  run:
+    - python >=3.6
+    - jigsaw >=0.9.12
+    - numpy
+    - pathlib
+
+test:
+  requires:
+    - matplotlib
```

Comment:
Can this be `matplotlib-base`

Reply:
Ah, good point. I believe it can be matplotlib-base. I'll change this.

Reply:
done!